### PR TITLE
Add support to mathjax style

### LINKED ISSUE #514: Latex/Mathjax Formula Support
When you ask GPT about math questions, sometimes in responds using Mathjax. Here is an example from a response from GPT 4:

`The user goes on to explain that if the variables are normalized in mean, matrix ( \frac{1}{m} A^T A ) is an estimator for the covariance between the variables. The notation ( s_{j1 j2} ) represents the estimated covariance between two variables ( a \cdot j1 ) and ( a \cdot j2 ), where ( E(a \cdot j1 a \cdot j2) ) is the expectation of the product of these two variables. The approximation ( \approx \frac{1}{m}\sum a_{j1} a_{j2} ) indicates that the covariance is approximated by the average of the product of corresponding values from these two variables.`

So, this request is for the ability for bettergpt to display notation such as: `( E(a \cdot j1 a \cdot j2) )` as `𝑎⋅𝑗⁢1`. The mathjax library will probably be helpful: https://www.mathjax.org/.

### PR DESCRIPTION
**Pull request by uoyniXnaH**
Resolve #514

### CODE REVIEW

**Comment by uoyniXnaH** [comment]
I would like add some information. I found that if I add rehype-katex and rehype-mathjax at the same time, the formulas may be rendered three times. So I keep using katex, only to replace delimiters like `\[` to `$$`, thus those mathjax expressions could be rendered by katex.

**Review by ztjhz — APPROVED** [review]

**Review by afkarfcom — APPROVED** [review]
